accio-main/src/main/java/io/accio/main/web/CacheResource.java

```diff
@@ -99,7 +99,7 @@ public void getDuckDBSettings(@Suspended AsyncResponse asyncResponse)
             asyncResponse.resume(Response.ok(cacheManager.getDuckDBSettings()).build());
         }
         catch (Exception e) {
-            asyncResponse.resume(Response.serverError().build());
+            asyncResponse.resume(asyncResponse.resume(e));
```

Comment:
one last comment, perhaps we can omit the try-catch block in this section. It seems that the AccioExceptionMapper is set up to catch the error and return a 500 response in its toResponse method. However, I'm not entirely certain about this, could you help me check this?

Reply:
Yes, If we throw the runtime exception from `cacheManager.getDuckDBSettings`, we can omit the try-catch.